### ISSUE TITLE
Tests are not passing with FillArrays@0.13.9

### ISSUE THREAD

**Issue by JordiManyer** [gridap/Gridap.jl#889]
This has been happening in new PRs (#888 , #886 ): CI is not passing very early on. 

After some testing, I am sure it's because of broadcasting for FillArrays. Tests are ok with version v0.13.7 but fail for v0.13.9. 

Error benchmark: 
```julia 
using FillArrays
using Test

a = Fill(ones(3,3),4)
a + a      # This fails!
a .+ a     # This is fine!
```
The line that fails works for v0.13.7. 

I am not sure if this is the desired behavior, but it seems to be the behavior now. 
Should I open an issue in FillArrays, or just fix our tests? 

@oriolcg @amartinhuertas @fverdugo

**Comment by amartinhuertas** [comment]
I would first add an issue in FillArrays to get an answer from them in regards to whether this is desired behaviour or a BUG.

**Comment by JordiManyer** [comment]
There is an issue in FillArrays already, I just wrote a comment. Let's see, I'll try to stay on top of it.

**Comment by JordiManyer** [comment]
This has been solved in FillArrays v0.13.10, tests seem to be passing now. Closing the issue.
